test/e2e/rte/metrics.go

```diff
@@ -102,18 +99,9 @@ var _ = ginkgo.Describe("[RTE][Monitoring] metrics", func() {
 		ginkgo.It("[EventChain] should have some metrics exported", func() {
 			rteContainerName, err := e2ertepod.FindRTEContainerName(rtePod)
 			gomega.Expect(err).ToNot(gomega.HaveOccurred())
-
-			stdout, stderr, err := k8se2epod.ExecWithOptions(f, k8se2epod.ExecOptions{
-				Command:            []string{"curl", fmt.Sprintf("http://127.0.0.1:%d/metrics", metricsPort)},
-				Namespace:          rtePod.Namespace,
-				PodName:            rtePod.Name,
-				ContainerName:      rteContainerName,
-				Stdin:              nil,
-				CaptureStdout:      true,
-				CaptureStderr:      true,
-				PreserveWhitespace: false,
-			})
-			gomega.Expect(err).ToNot(gomega.HaveOccurred(), "ExecWithOptions failed with %s:\n%s", err, stderr)
+			cmd := []string{"curl", fmt.Sprintf("http://127.0.0.1:%d/metrics", metricsPort)}
+			stdout, stderr, err := remoteexec.CommandOnPod(f.Ctx, f.K8SCli, rtePod, rteContainerName, cmd...)
```

Comment:
Let's dump the stdout at least initially. Something like
```
stdout, stderr, err := remoteexec.CommandOnPod(f.Ctx, f.K8SCli, rtePod, rteContainerName, "curl", fmt.Sprintf("http://127.0.0.1:%d/metrics", metricsPort))
klog.Infof("curl stdout BEGIN:\n%s\ncurl stdout END", stdout)
klog.Infof("curl stderr BEGIN:\n%s\ncurl stderr END", stderr)
```

Reply:
I actually thought about that but since it's done in the `Eventually` wouldn't it spam the logs badly?

Reply:
I dump the latest `stderr`\`stdout` in case of failure